include/ibamr/IBStrategy.h

```diff
@@ -225,6 +225,14 @@ class IBStrategy : public SAMRAI::mesh::StandardTagAndInitStrategy<NDIM>, public
     virtual bool getLagrangianStructureIsActivated(int structure_number = 0,
                                                    int level_number = std::numeric_limits<int>::max()) const;
 
+    /*!
+     * Get the ratio of the maximum point displacement of all the structures
+     * owned by the current class to the cell width of the grid level on which
+     * the structure is assigned. This value is usually used to determine if the
+     * Eulerian patch hierarchy needs to be regridded.
```

Comment:
```suggestion
     * Eulerian patch hierarchy needs to be regridded and/or if the assignment of Lagrangian data to Cartesian grid patches needs to be updated.
```

Reply:
No, this isn't true - `IBFEMethod` deals with that problem (reassociating Lagrangian structures with the Eulerian grid) in an independent way. See `IBFEMethod::preprocessIntegrateData()`.

Reply:
In particular: the displacement since the last regrid does not necessarily equal the displacement since the last patch-element association.

Reply:
Could you help future me by adding to this comment by pointing to where this is handled and how this is distinct from the re-association trigger?

Reply:
I am stuck in the past, when regridding was the only way to update the associations between patches and elements.

Reply:
That was only three months ago :)